README.md

```diff
@@ -14,147 +13,115 @@ Want to participate in [Hacktoberfest](https://hacktoberfest.com/)? All you have
-#### 3. Populate the sheet with some data.
-##### Data Schema
-    SPEAKERS                                                 SIGN UP
-    name : string                                            name  : string
-    title: string                                            email : string
+- **Create a google sheet**
+ 1. Go to https://docs.google.com/ and create a blank sheet
+ 2. Add name, title, team, location, shortBio, pronouns, imageLink headers in the sheet (make sure the order of headers is same as in the example sheet provided below).
+ 3. Populate the sheet with some data.
+    ``` 
+    Data Schema
+    name : string
+    title: string
     team: string
     location: string
     shortBio: string
     pronouns: string
     imageLink: string (url of hosted image)
-#### 4. Change the sheet name to "Speakers" (Sheet name "Speakers" is case sensitive). Add another sheet from add and change the sheet name to "signup" (Case sensitive 🙂).
+    ```
+  4. Change the sheet name to "Speakers" (Sheet name "Speakers" is case sensitive).
+  <br>
 
 ![](https://i.postimg.cc/MGPDVgkF/Sheet-Demo.jpg)
-### Example sheet 
+
+**Example sheet**: 
 https://docs.google.com/spreadsheets/d/1XgyHXaReTZ3Nq_r7QS18GDvqK_ht010QqnI6PXAnePA/edit?usp=sharing
 
+<br>
+
+- **Deploy App Script Web App**
+ 1. Open your google sheet.
+ 2. Click on Extensions tab.
+ 3. Click on App Script. 
 
-### 2. Deploy App Script Web App
 
-#### 1. Open your google sheet.
-#### 2. Click on Extensions tab.
-#### 3. Click on App Script.
 ![](https://i.postimg.cc/x8BPkmzp/App-Script.jpg)
-#### 4. Delete all the code from the editor (inside Code.gs file).
-####
-        // Get requests
+ 4. Delete all the code from the editor (inside Code.gs file).
+```
         function doGet(req) {
             if(req.parameters.sheetName == "Speakers") {
                 return getSpeakersData()
-            } else if(req.parameters.sheetName == 'Talks') {
-                return getTalksData()
             }
         }
 
-        // speaker requestHandler
         function getSpeakersData() {
-            var doc = SpreadsheetApp.getActiveSpreadsheet()
-            var sheet = doc.getSheetByName("Speakers")
-            var values = sheet.getDataRange().getValues()
-            
-            var output = []
-            for(var i=1; i<values.length; i++) {
-                var row = {}
-                row['name'] = values[i][0]
-                row['title'] = values[i][1]
-                row['team'] = values[i][2]
-                row['location'] = values[i][3]
-                row['shortBio'] = values[i][4]
-                row['pronouns'] = values[i][5]
-                row['imageLink'] = values[i][6]
-
-                output.push(row)
-            }
-
-            return ContentService.createTextOutput(JSON.stringify({speakers: output})).setMimeType(ContentService.MimeType.JSON)
-        }
-
-        // talk requestHandler
```

Comment:
I think you might need to make sure you've fetched the most recent version of `main` because this part is missing in your PR and it's very important to have!